packages/ia-activity-indicator/src/ia-activity-indicator.ts

```diff
@@ -0,0 +1,161 @@
+import { LitElement, html, css, property, CSSResult, TemplateResult } from 'lit-element';
```

Comment:
I thought we had the discussion and vote twice to not use TypeScript going forward.

Reply:
Ah, right. I'll hack together a solution instead of using sane data structures.